modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,58 @@
+= API Groups
```

Comment:
I would use this document as a reference for explaining API Groups 
https://docs.google.com/document/d/1ESY7h7Fuff_KttFGFvPUxNMfF4poeWXf5o8EcML0Jvk/edit#

Please be mindful of the deprecated term "API as Products"

Reply:
Done! Thanks!